hammr/commands/image/image.py

```diff
@@ -712,3 +693,20 @@ def cancel_image(self, image):
     def initialize_text_table(self, width):
         table = Texttable(width)
         return table
+
+    def get_uid_from_uri(self, uri, uid_type):
+        args = uri.split("/")
+        for index, arg in enumerate(args):
+            if arg == uid_type and index + 2 <= len(args):
+                return args[index + 1]
```

Comment:
It's a bit tricky to read this one, even if perfectly correct. Could avoid managing an index by using a boolean instead. E.g.:
```
boolean uid_type_found = false
for arg in enumerate(args):
  if uid_type_found:
    return arg
  else
    uid_type_found = arg == uid_type
```
What do you think?

Reply:
This a clean and clever way to do it, thank you !